### PR TITLE
[dashboard] minor improvements for email domains

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

We currently have 8000 entries on the server and we don't expect this to increase drastically, so server-side pagination is not necessary here. But the UI on the frontend should do pagination as otherwise the list and page is quite large.

## Related Issue(s)

Fixes #

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - se-admin-d58fa058f30</li>
	<li><b>🔗 URL</b> - <a href="https://se-admin-d58fa058f30.preview.gitpod-dev.com/workspaces" target="_blank">se-admin-d58fa058f30.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>
 
/hold

### CODE REVIEW

**Review by geropl — APPROVED** [review]
Code LGTM, tested and works :heavy_check_mark:

File: components/dashboard/src/admin/BlockedEmailDomains.tsx [hunk 1]
```diff
@@ -95,37 +120,39 @@ export function BlockedEmailDomainsList(props: Props) {
                             <input
                                 className="w-64 pl-9 border-0"
                                 type="search"
-                                placeholder="Search by URL RegEx"
+                                placeholder="Search by domain"
```
Comment (gtsiolis) [inline components/dashboard/src/admin/BlockedEmailDomains.tsx:123]: **nitpick**: Search placeholder, following other search inputs.

```suggestion
                                placeholder="Search Email Domains"
```

File: components/dashboard/src/admin/BlockedEmailDomains.tsx [hunk 2]
```diff
@@ -95,37 +120,39 @@ export function BlockedEmailDomainsList(props: Props) {
...
                 </div>
 
-                <Alert type={"info"} closable={false} showIcon={true} className="flex rounded p-2 mb-2 w-full">
-                    Search by email domain URL using <abbr title="regular expression">RegEx</abbr>.
-                </Alert>
                 <div className="flex flex-col space-y-2">
                     <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
-                        <div className="w-9/12">Domain URL (RegEx)</div>
+                        <div className="w-9/12">Domain</div>
                         <div className="w-1/12">Block Users</div>
```
Comment (gtsiolis) [inline components/dashboard/src/admin/BlockedEmailDomains.tsx:138]: **question**: Not the best place to comment about this but should we enable this option when adding a new Blocked Email Domain? Same goes for the Blocked Repositories. Both can be considered out of the scope of this PR though. Your call. 🏓 

| BEFORE | AFTER |
|--------|--------|
| Frame 1539 | Frame 1540 |

**Review by gtsiolis — COMMENTED** [review]
Posting below some minor UX comments, a bit late as this is already merged.

Cc @svenefftinge @geropl

File: components/dashboard/src/admin/BlockedEmailDomains.tsx (hunk 2 above)
Comment (svenefftinge) [reply components/dashboard/src/admin/BlockedEmailDomains.tsx:138]: Yes, would make sense. But also not super important as this is only used by internal admins

**Pull request merged**